hashicorp/boundary-ui
# refactor: 💡 move worker filter generator outside of rose::form

# Description

Refactor worker filter generator to be in a separate component file.
Before this change, the code editor and filter generator were nested within a form component, this PR extracts the editor and generator into its own file for use in a storage buckets form.




## Screenshots (if appropriate)

## How to Test

1. Validate the edit-egress-worker-filter & edit-ingress-worker-filter pages still work as expected.

## Checklist
